nsqadmin/nsqadmin.go

```diff
@@ -115,7 +115,7 @@ func (n *NSQAdmin) Main() {
 	n.Unlock()
 	httpServer := NewHTTPServer(&Context{n})
 	n.waitGroup.Wrap(func() {
-		http_api.Serve(n.httpListener, httpServer, "HTTP", n.opts.Logger)
+		http_api.Serve(n.httpListener, http_api.CompressHandler(httpServer), "HTTP", n.opts.Logger)
```

Comment:
I prefer this globally because the responses from some of the stats api endpoints could be pretty big and we want to compress as much as possible back to the client, (and json api responses will compress well). (and also as i recently realized I had a 250k response from nsqd `/stats` we may want to use this elsewhere too). Really, the only thing we wouldn't particularly want to compress is proxied graphite requests, but i think i'd rather address that with a future PR that adds mime type opt-out on the compression middleware.